.github/workflows/ci.yaml

```diff
@@ -11,6 +11,8 @@ on:
         required: false
         default: ''
         description: 'Branch that contains manifest and patches. Default is empty which uses GITHUB_SHA'
+  schedule:
```

Comment:
Does the `workflow_dispatch` section still need the `TRIAL_BRANCH` input?

Reply:
If TRIAL_BRANCH isn't used, then lines 52,60 aren't needed

Reply:
Removed